requirements/system/functional/server.tex

```diff
@@ -0,0 +1,29 @@
+\subsection{Central Server requirements}
+\subsubsection{Main features}
+\begin{enumerate}
+  \item[FRCS10] The central server shall distribute \glspl{job} among the available work machines using the scheduling algorithm.
```

Comment:
In the last meeting we agreed on calling the work machines. Work client or something like that. We should change it everywhere in the document.